sites/shared/components/workbench/inputs/measurement.mjs

```diff
@@ -11,16 +11,18 @@ import { measurementAsMm } from 'shared/utils.mjs'
  * m holds the measurement name. It's just so long to type
  * measurement and I always have some typo in it because dyslexia.
  */
-export const MeasurementInput = ({ m, gist, app, updateMeasurements, focus }) => {
+export const MeasurementInput = ({ m, gist, app, updateMeasurements, focus, optional }) => {
   const { t } = useTranslation(['app', 'measurements'])
   const prefix = app.site === 'org' ? '' : 'https://freesewing.org'
   const title = t(`measurements:${m}`)
 
   const isDegree = isDegreeMeasurement(m)
   const factor = useMemo(() => (isDegree ? 1 : gist.units == 'imperial' ? 25.4 : 10), [gist.units])
```

Comment:
I know it's not in the scope of the PR, but while you're here can you please fix these lint warnings? If fixing them messes everything up we can bump it to the workbench refactor

Reply:
I believe that I eliminated the `eslint` warnings in a way that is correct. I had to add more `useCallback()` hooks to deal with additional `eslint` warnings that popped up due to the changes.